package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "vscode-docker",
-    "version": "1.12.5-alpha",
+    "version": "1.13.0",
```

Comment:
Just for the record, this need to be updated to 1.14.0-alpha, but that can happen as a separate PR